docs/guides/on-demand-guides/index.mdx

```diff
@@ -3,4 +3,740 @@ title: On Demand
+};
+
+```
+
+### Response
+
+```
+{
+  "id": "4c21895e-2488-4c0a-9377-3369ff586ddd",
+  "hash": [
+    {
+      "hash": "6dad0a8e7a92d537c6305fe6ae1e1881",
+      "algorithm": "md5"
+    },
+    {
+      "hash": "6132f4e43472b56b4047b7ad018147056c0c2dba9306cd04504e9c8208e72257",
+      "algorithm": "sha256"
+    }
+  ],
+  "name": "New waterfall",
+  "size": 32407849,
+  "status": {
+    "phase": "ready",
+    "updatedAt": 1652901778931
+  },
+  "userId": "eada599f-3d58-499b-ba24-c7f3faf988de",
+  "createdAt": 1652901725324,
+  "updatedAt": 1652901778931,
+  "videoSpec": {
+    "format": "mp4",
+    "tracks": [
+      {
+        "fps": 23.976023976023978,
+        "type": "video",
+        "codec": "h264",
+        "width": 4096,
+        "height": 2048,
+        "bitrate": 21324821,
+        "duration": 12.011667,
+        "pixelFormat": "yuv420p"
+      },
+      {
+        "type": "audio",
+        "codec": "aac",
+        "bitrate": 253375,
+        "channels": 2,
+        "duration": 12.031667,
+        "sampleRate": 48000
+      }
+    ],
+    "duration": 12.031667
+  },
+  "playbackId": "4c21k7z2tjcdg8wt",
+  "playbackUrl": "https://livepeercdn.com/recordings/54aeff83-404c-490a-9afa-99296a932401/index.m3u8",
+  "downloadUrl": "https://livepeercdn.com/asset/4c21k7z2tjcdg8wt/video"
+}
+```
+
+## Delete an asset
+
+### Request
+
+```JavaScript
+
+const deleteAsset = async () => {
+  const response = await axios.delete(
+    "https://livepeer.studio/api/asset/{id}",
+    {
+      headers: {
+        Authorization: `Bearer ${process.env.LP_API_KEY}`,
+      },
+    }
+  );
+  JSON.stringify(response.data);
+};
+```
+
+### Response
+
+When the asset has been successfully deleted, the response should return a 204
+
+## Export an asset to IPFS
+
+This method allows for exporting an asset imported or uploaded from Livepeer to IPFS.
+
+### Request
+
+Provide the $ASSET_ID of the asset, this can be found using the List Asset method.
+
+```JavaScript
+
+const exportAsset = async () => {
+  const response = await axios.post(
+    "https://livepeer.studio/api/asset/$ASSET_ID/export",
+    {
+      headers: {
+        Authorization: `Bearer ${process.env.LP_API_KEY}`,
+        "Content-Type": "application/json",
+      },
+      data: JSON.stringify({
+        ipfs: {},
+      }),
+    }
+  );
+  console.log(JSON.stringify(res.data));
+};
+```
+
+### Response
+
+The this example has the response show that the status is pending.
+
+```JSON
+{
+  "task": {
+    "id": "8b0cd796-8be1-412b-b1dc-8251a35ad2e7",
+    "createdAt": 1652903903963,
+    "type": "export",
+    "inputAssetId": "87289307-b60f-432b-984a-8ede174fa152",
+    "userId": "eada599f-3d58-499b-ba24-c7f3faf988de",
+    "params": {
+      "export": {
+        "ipfs": {}
+      }
+    },
+    "status": {
+      "phase": "pending",
+      "updatedAt": 1652903903963
+    }
+  }
+}
+Check back with https://livepeer.studio/api/task/$TASK_ID to get the status update.
```

Comment:
Check back with `https://livepeer.studio/api/task/$TASK_ID` to get the status update.